CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 2.5 (in-progress)
 ================
-* [#163](https://github.com/dblock/oshi/pull/163): Update getFileStores() method to include mount point [@henryx](https://github.com/henryx).
-* [#165](https://github.com/dblock/oshi/pull/165): Added system-wide file descriptor counts - [@cholland1989](https://github.com/cholland1989).
+* [#163](https://github.com/dblock/oshi/pull/163): Update getFileStores() method to include mount point [@henryx](https://github.com/henryx),
+* [#165](https://github.com/dblock/oshi/pull/165): Added system-wide file descriptor counts - [@cholland1989](https://github.com/cholland1989)
 * [#168](https://github.com/dblock/oshi/pull/168): Switched WMI queries to use COM - [@dbwiddis](https://github.com/dbwiddis).
 * [#171](https://github.com/dblock/oshi/pull/171): Added detailed process statistics - [@dbwiddis](https://github.com/dbwiddis).
+* [#176](https://github.com/dblock/oshi/pull/176):  Fix issue #169 - [@henryx](https://github.com/henryx).
```

Comment:
Make this comment stand on its own.  "Eliminate Swing threads in WindowsFileSystem" or something similar. Also notice the periods added and comma removed on earlier changelogs.